### ISSUE TITLE
Possible Python GUI hooks

### ISSUE THREAD

**Issue by Circu1tI0N3rd** [PythonTryHard/BKSchedule_Rewrite#5]

**Comment by PythonTryHard** [comment]
What do you mean by `pyWebView` interfere with `requests`? And which one would be the easiest to start from absolute ground zero, **and** able to compile to Android without much of a hassle?

**Comment by Circu1tI0N3rd** [comment]
Literally pyWebView is a web browser in a nutshell and yet it used the backbone urllib. Thus it also uses native system graphical wrappers aka windowsform, objectivec,... but NOT Android graphic backbone

**Comment by PythonTryHard** [comment]
Got some worrying result from [StackOverflow](https://stackoverflow.com/questions/17721796/tkinter-to-android-translation) though

**Comment by Circu1tI0N3rd** [comment]
True. I've already tried to stay away from TK and yet currently TK is used purely for testing. I'm using TK cuz there's a GUI designer for it while Kivy has a obsolete one which I did try to get it working but still doesn't. Yet there's code to make a table in TK that worked great but not for longterm (screenshot).

![image](https://app.gitkraken.com/api/glo/boards/5dd823a6e702e8000fafeec6/attachments/5def4dfc5b8585000f22a416)

**Comment by PythonTryHard** [comment]
So we have the hard to implement solution but has a GUI designer, or the (relatively) easy to implement solution but doesn't have a GUI designer. This is is bloody awful
